selfdrive/car/hyundai/interface.py

```diff
@@ -133,7 +133,16 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       tire_stiffness_factor = 0.385
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
-    elif candidate in [CAR.IONIQ, CAR.IONIQ_EV_LTD]:
+    elif candidate in [CAR.IONIQ]:
```

Comment:
Should this be named better? `IONIQ_EV_2020plus`or something?

Reply:
It accounts for all ioniq models 2020 from what I know

Reply:
Then the read me update should probably list the rest of the Ioniqs too, vs just the EV.

On second look, it appears this PR doesn't include an update for the readme.